lib/schemas/CssColor.json

```diff
@@ -0,0 +1,221 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "kendo-cli://schemas/CssColor.json",
```

Comment:
Shouldn't that be a valid URL?

Reply:
yep, it should be. However, I don't know what the real URL should be, so I am leaving it with a pseudo URL fragment that we are to replace.